adapt-and-extend/theming/resources.rst

```diff
@@ -356,9 +356,38 @@ The options are :
 Diazo frontend - barceloneta backend
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Using diazo rules you can define a frontend and a backend separatelly defining which bundles you want to load.
+Using diazo rules you can theme the frontend of your site how you like, and use the default Barceloneta theme for the backend.
```

Comment:
Would the `<replace/>` workaround in the accepted answer not work for a lot of (possibly all) scenarios where you would need to use inline xsl?

Is this also not a similar problem regardless of whether you are using this backend template? In the [theming guide](https://plone-theming-with-diazo.readthedocs.org/en/latest/snippets_diazo/rules/index.html) we see:

```
 <!-- theme wrapper, to prevent theming ZMI -->
 <rules css:if-content="#visual-portal-wrapper">
```

so inline xsl directives would also need to be defined outside the nested `<rules>`, potentially causing issues with the zmi.